src/components/HelpModal.js

```diff
@@ -0,0 +1,22 @@
+import ReactModal from 'react-modal'
+import '../css/HelpModal.css'
+
+function HelpModal({ isOpen, close}) {
+    return (
+        <ReactModal className="modal" isOpen={isOpen} parentSelector={() => document.querySelector('.Game')} contentLabel="Instructions">
+            <div className='help'>
```

Comment:
This looks great! Nice job with these instructions. One note: it might be worth `text-align: left` on the p tags -- multi-line centered text apparently is an issue for dyslexic users.

Reply:
I second this. I appreciate the important instructions being bolded, though. That helps a lot with quick comprehensibility.

Reply:
Implemented!